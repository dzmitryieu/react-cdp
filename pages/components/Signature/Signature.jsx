import React from 'react';
import styles from './Signature.scss';

const Signature = () => (
  <footer className={styles.footer}>
    <p className={styles.name}>netflixroulette</p>
  </footer>
);

export default Signature;
