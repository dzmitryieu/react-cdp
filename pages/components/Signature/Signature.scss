@import "../../utils/constants.scss";

.footer {
  height: 130px;
  width: 100%;
  box-shadow: 0 10px 40px rgba(0, 0, 0, 0.04);
  background-color: $dark-gray;
  text-align: center;
  font-family: $helvetica;
  display: flex;
  justify-content: flex-start;
  flex-direction: row;
  align-items: center;
  margin-top: -130px;
  padding: 0 60px;

  .name {
    color: $light-red;
    font-size: 25px;
    letter-spacing: 2px;
    font-weight: 800;
  }
}
