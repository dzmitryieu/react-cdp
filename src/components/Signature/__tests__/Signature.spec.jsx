import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Signature from '../index';

const renderer = new ShallowRenderer();

describe('Signature Snapshot', () => {
  test('renders', () => {
    const component = renderer.render(
      <Signature />,
    );
    expect(component).toMatchSnapshot();
  });
});
