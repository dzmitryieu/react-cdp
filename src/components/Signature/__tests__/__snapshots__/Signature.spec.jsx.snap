// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Signature Snapshot renders 1`] = `
<footer
  className="footer"
>
  <p
    className="name"
  >
    netflixroulette
  </p>
</footer>
`;
